core/src/com/unciv/logic/automation/unit/HeadTowardsEnemyCityAutomation.kt

```diff
@@ -132,11 +132,12 @@ object HeadTowardsEnemyCityAutomation {
         val tileToMoveTo =
             unitDistanceToTiles.asSequence()
                 .filter {
-                    it.key.aerialDistanceTo(closestReachableEnemyCity) <=
-                        unitRange && it.key !in tilesInBombardRange
+                    it.key.aerialDistanceTo(closestReachableEnemyCity) >= unitRange 
```

Comment:
The current logic tries to find a tile for which the unit is within firing range, if that requirement can be dropped (which it is due to the new ordering at line 140), then this comparison should be dropped altogether.